### ISSUE TITLE
Locast Captcha issue

### ISSUE THREAD

**Issue by n9mdh** [wouterdebie/locast2tuner#75]
Locast's captcha requirement seems to be preventing locast2tuner from logging in, based on identical reported problems in the locast2plex git -- logins refused, yet accessing locast through their app or website is fine.

Looking at the latest credentials.rs, it does appear that a value ("foo") is present for the captcha, which negates the solution in the locast2plex conversation.

Any ideas? 

I'm running in a Debian environment, yes everything is updated, and locast2tuner was running fine business until a few days ago.

**Comment by eiddor** [comment]
Just tested with the latest build (0.1.48) and it's logging in fine here.

**Comment by jg-24** [comment]
locast2tuner 0.1.46 fixed the login issue for me on Aug 4th.  I just upgraded to 0.1.48 and it logged in fine.  

I did have a weird issue with Plex not playing any live TV even though the tuner showed connected, but there was a Plex upgrade at the same time.  Double-checked locast2tuner config to make sure no new settings were added and restarted Plex and it all works fine now -- so maybe there's something like that going on with your system?

**Comment by wouterdebie** [comment]
No issues here either.. And the difference between locast2plex and locast2tuner is insignificant. The captcha isn't used, so locast2tuner provides "foo" as a value, while locast2plex provides "locast2plex".

@n9mdh can you provide the actual version you're using and perhaps some log?

**Comment by n9mdh** [comment]
The version involved is the current, 0.1.48.

During the day today, I removed the service and Deb package, then installed again, with no luck. Trimming down the config to bare bones minimum still provided no joy. The error was the panic message that the login had failed.

The machine involved is just a Raspberry Pi 4, so nuking the OS and starting over was pretty simple. After installing locast2tuner, I restored the archive of the config file I had been using before, and on reboot the service came up and was able to login successfully.

Around the time of the 0.1.48 update, something must have gotten corrupted on the machine. Totally weird, as nothing else running on the machine was messed up. As a rule I never believe in coincidences, but this appears to be one crazy coincidence.
